app/router.js

```diff
@@ -1,7 +1,7 @@
 import Ember from 'ember';
 import config from './config/environment';
 
-var Router = Ember.Router.extend({
+let Router = Ember.Router.extend({
```

Comment:
'let' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).